spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -114,15 +114,17 @@ def get_parser():
     parser.add_argument('-include-list',
                         help='Optional space separated list of subjects to include. Only process '
                         'a subject if they are on this list. Inclusions are processed before exclusions. '
-                        'Cannot be used with `include`.', nargs='+')
+                        'Cannot be used with `include`. \n'
+                        'Example: \'-include-list sub-001 sub-002\'', nargs='+')
     parser.add_argument('-exclude',
                         help='Optional regex used to filter the list of subject directories. Only process '
                         'a subject if they do not match the regex. Exclusions are processed '
                         'after inclusions. Cannot be used with `exclude-list`')
     parser.add_argument('-exclude-list',
                         help='Optional space separated list of subjects to exclude. Only process '
                         'a subject if they are not on this list. Inclusions are processed before exclusions. '
-                        'Cannot be used with either `exclude`.', nargs='+')
+                        'Cannot be used with either `exclude`. \n'
```

Comment:
While we're here, we can fix this small typo too:

```suggestion
                        'Cannot be used with `exclude`. \n'
```